app/src/main/AndroidManifest.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
Also please change the title of your PR. It should not just be "Brandon/dev"

Reply:
Renamed it